build-infrastructure/README.md

```diff
@@ -42,13 +42,26 @@ The directory structure that is expected is as follows,
 
 ```
 buildspecs/
-|- build.yml
+|- merge-build.yml
```

Comment:
Do we add a bullet for pr-build.yml here, since it will also live here?

Reply:
Yeah, we should! I missed that

Reply:
Going to address this in #3205.